src/cases/containers/case-viewer-container/case-viewer-container.component.spec.ts

```diff
@@ -1,29 +1,101 @@
 import { Component, Input } from '@angular/core';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, fakeAsync, TestBed } from '@angular/core/testing';
```

Comment:
_Nitpick:_
You've introduced `fakeAsync` but it doesn't look like it's being used anywhere in here.